get-started/index.md

```diff
@@ -62,7 +65,291 @@ the specifics of how to deploy your Cryostat instance. Continue to [Setup](#setu
+          ports:
+            - containerPort: 9977
+              protocol: TCP
+          resources: {}
+      restartPolicy: Always
+status: {}
+```
+
+Port number `9977` is the default HTTP port that the Agent exposes for its internal webserver that services Cryostat requests. The `CRYOSTAT_AGENT_AUTHORIZATION` value is particularly
+noteworthy: these are the credentials that the Agent will include in API requests it makes to Cryostat to advertise its own presence. You should create an OpenShift Service Account for
+this purpose and replace `abcd1234` with the base64-encoded authentication token associated with the service account. For testing purposes you may use your own user account's
+authentication token, for example with `oc whoami --show-token`.
+
+Finally, create a Service to enable Cryostat to make requests to this Agent:
+
+```yaml
+apiVersion: v1
+kind: Service
+...
+spec:
+  ports:
+    - name: "cryostat-agent"
+      port: 9977
+      targetPort: 9977
+...
+```
+
+More details about the configuration options for the Cryostat Agent [are available here](https://github.com/cryostatio/cryostat-agent/blob/main/README.md#configuration).
+
+#### [Using JMX](#using-jmx)
+Cryostat is also able to use Java Management Extensions (JMX) to communicate with target applications. This is a standard JDK feature that can be enabled by passing JVM
+flags to your application at startup. A basic and insecure setup suitable for testing requires only the following three flags:
+
+```
+-Dcom.sun.management.jmxremote.port=9091
+-Dcom.sun.management.jmxremote.ssl=false
+-Dcom.sun.management.jmxremote.authenticate=false
+```
+
+[comment]: # TODO explain how to configure SSL and auth for JMX, or link to external docs
+
+In a real scenario you should enable both SSL and authentication on your application. You can then [trust the certificate](/guides/#add-a-trusted-certificate)
+and [store the credentials](/guides/#store-jmx-credentials).
+
+Depending on your application or its framework, you may set these flags directly in a `Dockerfile` entrypoint, an environment variable, or similar. This may or
+may not require a container image rebuild, and it will require the container to be restarted. Once this is done the application container will be listening for
+incoming JMX connections on port 9091. Let's assume it can be done by setting an environment variable, so we only need to mofify our Deployment:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+...
+spec:
+  ...
+  template:
+    ...
+    spec:
+      containers:
+        - name: sample-app
+          image: docker.io/myorg/myapp:latest
+          env:
+            - name: JAVA_OPTS
+              value: "-Dcom.sun.management.jmxremote.port=9091 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
+            ...
+```
+
+Next, we need to configure an OpenShift Service to expose this port for cluster-internal traffic, so that Cryostat can see
```

Comment:
I think just `Service` (need to be inline-code) is fine right? Since that should be the same for k8s?